### PR TITLE
[Feature] Gulp: Validate node.js version before running

### PR DESCRIPTION
**Pull request by d-simon**
This change adds a node.js version check before running any gulp tasks. The version check is based on the information in the package.json: 

```json
  "engines": {
    "node": "6.7.0"
  }
```

### CODE REVIEW

**Comment by backflip** [comment]
Could you rebase this on develop? There was some force-pushing happening for v5.0.0.

yolo

**Comment by 0xfffxfff** [comment]
"Rebase"

really_a_pony_version_of_a_meme_some_bronies_really_2fb4eedd3de7d2f02ac5b5b683891f6d-1

**Comment by d-simon** [comment]
"Rebase"
really_a_pony_version_of_a_meme_some_bronies_really_2fb4eedd3de7d2f02ac5b5b683891f6d-1

**Comment by backflip** [comment]
`alias rebase="git reset --hard origin/develop && git cherry-pick 67319ca6ed0c8d6d174f4899c49763e2bd4f6765 && git push -f"`

How hard can it be?

**Comment by backflip** [comment]
LGTM

**Review by orioltf — CHANGES_REQUESTED** [review]
@d-simon would you mind doing the rebase in your branch, please? Nothing else is missing to get green light here! #gogogo

**Comment by d-simon** [comment]
@orioltf great to hear. It saved one of our devs many times.
IIRC there is a Rebase&Merge button in gh now. No? 😜 Otherwise I can rebase it tomorrow.

**Comment by d-simon** [comment]
Ah I didn't see the conflicts from mobile, my bad.
I had to remove the `npm-shrinkwrap.json` because I wasn't able to get into a conflict free state. @orioltf @backflip Is there a reason we still keep a shrinkwrap json file instead of `yarn.lock`?

**Comment by orioltf** [comment]
@d-simon You can have a look at issue #12 However, I bet your comment was meant to be an upcoming PR, right?! 😝

**Comment by backflip** [comment]
Replaced with https://github.com/unic/estatico/pull/44 with a regenerated `npm-shrinkwrap.json`

**Pull request closed without merging**
